10-zwave.js

```diff
@@ -410,7 +410,16 @@ module.exports = function(RED) {
 					) {
 						if (debug) console.log('attempting direct call to OpenZWave API: %s(%s)', msg.topic, payload);
 						try {
-							ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var retval = ozwDriver[msg.topic].apply(ozwDriver, payload);
+							var newmsg = {'topic': msg.topic};
+							if (retval) {
+								if (debug) console.log('Got return value, sending as payload');
+								newmsg.payload = retval;
+							} else {
+								if (debug) console.log('No return value but call successful, sending empty array');
+								newmsg.payload = [];
+							}
+							node.send(newmsg);
```

Comment:
Wont this addition cause the zwave-out node to send out messages to NR flows as well?